src/Libraries/Microsoft.Extensions.AI.Abstractions/Contents/JsonSchemaInferenceOptions.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.Extensions.AI;
+
+/// <summary>
+/// An options class for configuring the behavior of <see cref="JsonFunctionCallUtilities"/> JSON schema inference functionality.
+/// </summary>
+public sealed class JsonSchemaInferenceOptions
+{
+    /// <summary>
+    /// Gets the default options instance.
+    /// </summary>
+    public static JsonSchemaInferenceOptions Default { get; } = new JsonSchemaInferenceOptions();
+
+    /// <summary>
+    /// Gets a value indicating whether to include the type keywork in inferred schemas for .NET enums.
```

Comment:
```suggestion
    /// Gets a value indicating whether to include the type keyword in inferred schemas for .NET enums.
```
Also, should `type` be `$type`?

Reply:
It's just `type` in JSON schema world.